test/chi/chi-vue/data-table.pug

```diff
@@ -7,38 +7,47 @@ h3 Base
   #base
```

Comment:
Indentation is correct here, can you do the same for data-table-portal.pug aswell, please?